src/server/modes/ctf/maintenance/chat.ts

```diff
@@ -46,6 +52,18 @@ export default class GameChat extends System {
     this.framesPassed = 0;
   }
 
+  protected static isHelpCommand(msg: string): boolean {
+    if (msg.charAt(0) !== '#' || msg.length > 5) {
+      return false;
+    }
+
+    if (msg === '#help') {
```

Comment:
It's better to make the check case insensitive like isAttack. Bots also respond to #Help, #HELp, etc.

Reply:
done